docs/compliance/dist/system-security-plans/ato/sc-7.4.md

```diff
@@ -1,11 +1,8 @@
 ---
 implementation-status:
-  - c-not-implemented
+  - c-partially-implemented
```

Comment:
Is it the case that cloud.gov is not using separate networks for their control plane or is it the case that we don't have enough information to say?